NDTensors/src/lib/Sectors/src/category_definitions.jl

```diff
@@ -0,0 +1,183 @@
+import HalfIntegers: Half
+
+#
+# Group U₁
+#
+
+struct U1 <: AbstractGroup
+  n::Half{Int}
+end
+
+label(u::U1) = u.n
+
+dimension(::U1) = 1
+
+trivial(::Type{U1}) = U1(0)
+
+fusion_rule(::Type{U1}, n1, n2) = (n1 + n2,)
```

Comment:
I think it would be better to split group/category definitions into different files.

Reply:
Done